.github/workflows/docker.yml

```diff
@@ -0,0 +1,82 @@
+name: Docker
+
+on:
+  push:
+    branches:
+      - main
+      - master
```

Comment:
I added `master`, as I am using it on my fork. But I am ok to remove it, let me know your preference.

Reply:
It's fine to keep both. The only reasons I know of to restrict them are to avoid being charged for too many build hours and to avoid being charged too much for container images storage. I don't think I'm in danger of either of those — these images are like 10MB and are built about 4 times every few months.